### PR TITLE
Preonboard readme docs

### PR DESCRIPTION
**Pull request by john-sifive**
Adds instructions to README.md for creating Scribble PIO documents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -800,3 +801,155 @@ global def pio16DUT =
...
+publish scribbleDirectories = simplify "../../docs/scribble", Nil
+```
+
+### Creating an Onboarding document.
+At this point, we can create the first draft of the PIO Onboarding document. Type the following command.
+```
+wake makeOnboardingDocument pioDUT
+```
+The Onboarding document is a short test document which displays all the sections we've written so far. 
+It is created in the directory `build/api-generator-sifive/pioDUT/documentation`.
```
Comment (richardxia) [inline README.md]: I think it's worth describing the file types that are produced here. Specifically, that there is an `.html` file that the user can directly view in their web browser, and that there is an `.adoc` file that shows the AsciiDoc markup.

File: README.md [hunk 2]
```diff
@@ -800,3 +801,155 @@ global def pio16DUT =
...
+If the block has registers, this chapter should invoke `RegisterMap` 
+to display a register map along with descriptions of all the register fields.
+(See [Scribble Test Socket](https://github.com/sifive/scribble-testsocket-sifive) for more information on `RegisterMap`.)
+
+For pioDUT, we are going to create the main `Programming` section which invokes `RegisterMap`, 
+and we're going to create a subsection called `InstanceTable` which displays the configurable values of each PIO instance.
+
+First, copy the following text into the main section `Programming.jinja2`.
+```
+{%  set registers = RegisterMap(scope) %}
```
Comment (richardxia) [inline README.md]:
```suggestion
{% set registers = RegisterMap(scope) %}
```

Sorry to nitpick about this, but let's remove the second space character before the word `set` here. I'll remove it from the actual Jinja template file in the `master` branch.

**Review by richardxia — APPROVED** [review]
This looks great! Thanks for writing this up!

File: README.md (hunk 2 above)
Comment (john-sifive) [reply README.md]: Thanks for proofreading! It really helps.

**Pull request merged**